rosdep/python.yaml

```diff
@@ -356,6 +356,33 @@ python-argparse:
       packages: []
     zesty:
       packages: []
+python-attrs:
+  debian: [python-attr]
+  fedora: [python2-attrs]
+  gentoo: [dev-python/attrs]
+  ubuntu:
+    artful:
+      packages: [python-attr]
+    bionic:
+      packages: [python-attr]
+    trusty:
+      pip:
+        packages: [attrs]
+    xenial:
+      packages: [python-attr]
+python-attrs-pip:
+  debian:
+    pip:
+      packages: [attrs]
+  fedora:
+    pip:
+      packages: [attrs]
+  gentoo:
```

Comment:
can you please remove gentoo from this list ? pip and gentoo don't play along very well from my understanding and as gentoo is a rolling distribution it (most of the time) provide the latest version of the packages